### PR TITLE
add no joint tenant radio button

### PR DESCRIPTION
**Pull request by LBHABEBIC**
# What?

# Why?

# Notes

# Next steps

### CODE REVIEW

**Review by Gweaton — COMMENTED** [review]
Does this need to be dynamic, like @erbridge's comment on the card - i.e. only added to the list if the tenant isn't marked as present?

If not - looks good!

**Comment by erbridge** [comment]
What I meant was that it's not relevant that a joint tenant wasn't present if there's only one tenant. I don't think that blocks this getting merged, though.

File: steps/id-and-residency/id.tsx [hunk 1]
```diff
@@ -48,6 +48,10 @@ const idTypeRadios = [
     label: "Unable to verify ID",
     value: "no id",
   },
+  {
+    label: "Joint tenant not present",
+    value: "no joint tenant",
```
Comment (erbridge) [inline steps/id-and-residency/id.tsx:53]:
```suggestion
    label: "Tenant not present",
    value: "not present",
```

**Comment by LBHABEBIC** [comment]
> What I meant was that it's not relevant that a joint tenant wasn't present if there's only one tenant. I don't think that blocks this getting merged, though.

Hi F. I spoke to Gill this morning and she wants the radio button to appear only for the tenants who are not present. So now my question is how do we get it to pop off the last radio button if the tenant is not present? Should this be part of a render when? render when one or more tenants are not present?

**Review by erbridge — APPROVED** [review]

**Pull request merged**